src/newSeed/newSeed.html

```diff
@@ -1,14 +1,14 @@
 <template>
   <require from="../resources/elements/circledNumber/circledNumber.html"></require>
 
-  <div class="newSeedContainer">
-    <div class="sideBarDropDownContainer">
+  <div class="newSeedContainer ${currentStage.settings.stageNumber === 7 ? 'stage7' : ''}">
```

Comment:
can you please put the `router.currentInstruction.config.name` as the class name for every seed, rather then special casing the seventh?